packages/web/src/scss/components/Icon/README.md

```diff
@@ -0,0 +1,15 @@
+# Icon
+
+Use Icon component to display an icon from our icon set. This icon set is
+exported from Supernova into separate `spirit-icons` package. So make sure you
+install it first and then copy the icons into your project.
```

Comment:
> So make sure you install it first and then copy the icons into your project.

That should not be necessary, `spirit-icons` are in the dependencies of this project. Or do I get it wrong?

Reply:
Removed